lib/pure/times.nim

```diff
@@ -1149,11 +1137,14 @@ proc `-`*(time: Time, interval: TimeInterval): Time =
   else:
     toTime(time.local - interval)
 
-proc `-=`*(time: var Time, interval: TimeInterval) =
-  ## Modifies `time` by subtracting `interval`.
-  ## If `interval` contains any years, months, weeks or days the operation
-  ## is performed in the local timezone.
-  time = time - interval
+proc `+=`*[T, U: TimesMutableTypes](a: var T, b: U) =
+  a = a + b
+
+proc `-=`*[T, U: TimesMutableTypes](a: var T, b: U) =
+  a = a - b
+
+proc `*=`*[T, U: TimesMutableTypes](a: var T, b: U) =
+  a = a * b
 
```

Comment:
You can't multiply a `TimesMutableTypes` by another `TimesMutableTypes`. Thats why in my proposal, i only restricted the first generic type to types defines in times.nim. This restriction is enough to avoid the ambiguity with the mutators in system.nim. The remaining validation can be left to the operations defined in times.nim

Reply:
hm. Valid point. But only for *=. I think other procs should remain as is

Reply:
done